frontend/app/src/premium/premium-apis.ts

```diff
@@ -41,22 +41,24 @@ import { useStatisticsStore } from '@/store/statistics';
 import { One } from '@/utils/bignumbers';
 
 export const assetsApi = (): AssetsApi => {
-  const { assetInfo, assetSymbol, tokenAddress } = useAssetInfoRetrieval();
+  const { assetInfo, assetSymbol, tokenAddress, getIdentifierBySymbol } =
+    useAssetInfoRetrieval();
 
   const { getNftDetails } = useNftAssetInfoStore();
 
   return {
-    assetInfo: identifier => {
+    assetInfo: (identifier: string) => {
```

Comment:
Would it be to much work inside components to change these methods to receive a MaybeRef and then return a computed?

Reply:
No problem, it's easy

Reply:
VueUse should already be provided in the components